core/src/game_state.rs

```diff
@@ -170,6 +170,18 @@ impl Default for BidPolicy {
     }
 }
 
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Serialize, Deserialize)]
+pub enum BonusLevelPolicy {
```

Comment:
I don't actually think this needs to be optional; I just never got around to implementing it. Might make sense to just change the default behavior here.

Reply:
let's keep it an option if it is OK with you as there are people think that landlord team gets into this situation mostly by being careless, and should not deserve a bonus :)

I will change the default to bonus, however